controllers/operands/kubevirt.go

```diff
@@ -45,6 +46,12 @@ const (
 	DefaultARM64EmulatedMachines = "virt*"
 )
 
+const (
+	// Needs to align with the NAD that will be deployed by CNAO
+	primaryUDNNetworkBindingName      = "primaryUserDefinedNetwork"
+	primaryUDNNetworkBindingNamespace = "default"
```

Comment:
why in the default ns?

Reply:
In order for the NetworkAttachmentDefinition to be available to all VMs on all namespaces, it needs to be deployed on `default` namespace.
For the dev-preview we think this is a good enough approach, but IMO in the future we will need to think of other ways.

Also note that HCO I changed the PR since when we talked and not it is not not deploying the NetworkAttachmentDefinition (will happen on CNAO)